portfolio/src/main/webapp/script.js

```diff
@@ -25,4 +25,15 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+//   document.write("\n");
+  // TBH I don't know if this is what you mean by trailing new line
```

Comment:
No. I meant that when you edit files, don't save a file without having pressed return at the end of the last line.